src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -102,6 +102,8 @@ class LayoutLMv2FeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
             Only has an effect if :obj:`do_resize` is set to :obj:`True`.
         apply_ocr (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether to apply the Tesseract OCR engine to get words + normalized bounding boxes.
+        ocr_lang (:obj:`Optional[str]`, `optional`, defaults to :obj:`None`):
+            Which language model the Tesseract OCR engine uses. English is used by default.
```

Comment:
Maybe you can add here that this should be a specific language code, like 'fra' for French.

A list of supported ones can be found here: https://tesseract-ocr.github.io/tessdoc/Data-Files-in-different-versions.html